src/Views/gagepage.html

```diff
@@ -240,7 +240,7 @@ <h4 class="modal-title">StreamStats Gage Page</h4>
                         <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.heatChartConfig">
                             <highchart id="chart2" config="vm.heatChartConfig"></highchart>
                         </div>
-                    </center>
+                    </center> -->
```

Comment:
Remember to uncomment for the final pull request 

```suggestion
                    </center>
```